### PR TITLE
FAB 2397 End-to-End Example Demonstrating Python SDK

### PR DESCRIPTION
**Pull request by bpandey-CS**
A possible solution to https://cognitivescale.atlassian.net/browse/FAB-2397 which aims at demonstrating an end-to-end example using the cortex-python (v6) SDK.

### CODE REVIEW

**Review by dvasani-CS — APPROVED** [review]

File: EndToEndExample/actions/train/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pandas
+numpy
+cortex-python==6.0.2a4
```
Comment (svangapally-cs) [inline EndToEndExample/actions/train/requirements.txt:3]: Update this to use latest stable version 6.0.3

File: EndToEndExample/actions/predict/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+pandas
+numpy
+cortex-python==6.0.2a3
```
Comment (svangapally-cs) [inline EndToEndExample/actions/predict/requirements.txt:3]: Same here

Comment (svangapally-cs) [reply EndToEndExample/actions/predict/requirements.txt:3]: Better to add the versions for all the added requirements to make sure that the code doesn't break with other versions.

**Review by svangapally-cs — APPROVED** [review]

File: EndToEndExample/actions/train/requirements.txt (hunk 1 above)
Comment (bpandey-CS) [reply EndToEndExample/actions/train/requirements.txt:3]: I've been using this alpha release specifically to use SkillClient

File: EndToEndExample/actions/predict/requirements.txt (hunk 2 above)
Comment (bpandey-CS) [reply EndToEndExample/actions/predict/requirements.txt]: done

**Review by dvasani-CS — APPROVED** [review]

**Pull request merged**